postgres/osmap.yaml

```diff
@@ -39,12 +50,6 @@ RedHat:
 
   pkg: postgresql{{ release }}-server
   pkg_client: postgresql{{ release }}
-  conf_dir: /var/lib/pgsql/{{ repo.version }}/data
-  service: postgresql-{{ repo.version }}
-
-  prepare_cluster:
-    command: initdb --pgdata='{{ data_dir }}'
-    test: test -f '{{ data_dir }}/PG_VERSION'
```

Comment:
This whole section is here for the purpose: the values should be set like this only when `use_upstream_repo` has been set to `True`.

Reply:
The state "postgres.server.postgresql-cluster-prepared" fails unless "prepare_cluster" stanza is defined somewhere for each distro.

>
postgresql-cluster-prepared:
  cmd.run:
    - name: {{ postgres.prepare_cluster.command }}
    - cwd: /
    - runas: {{ postgres.prepare_cluster.user }}
    - env: {{ postgres.prepare_cluster.env|default({}) }}
    - unless:
      - {{ postgres.prepare_cluster.test }}
    - require:
      - pkg: postgresql-server
>

Reply:
My bad - I see this in defaults.yaml